lib/material/material.flow

```diff
@@ -1698,6 +1698,8 @@ export {
 				MTabsNoCapitalization();
 				// Changes the color of the tab text when selected
 				MTabChangeIndicatorColor();
+				// Changes tab's header color on selected
+				MTabSelectedBackgroundColor(color : MColor, opacity : double);
```

Comment:
Let's make it  MTabSelectedBackgroundStyle(style : Transform<TGraphicsStyle>) to be more general-purpose